### ISSUE TITLE
Doesn't seem to scan few cards 

### ISSUE THREAD

**Issue by jagan27101986** [vkeepe/card.io#10]
The app doesn't seem to scan few cards such as AMEX, HSBC debit cards and other cards. Could you please advise what needs to be done for the same? The issue is presented both in IOS and Android

**Comment by seatechdev** [comment]
Can you provide a card number?  I can manually enter and check.

**Comment by ba0f3** [comment]
card.io can only scan cards that have numbers w/ silver overlay

**Comment by jagan27101986** [comment]
card.io doesn't scan card numbers  with black overlay for Amex cards ?

**Comment by ba0f3** [comment]
Yes, i faced same problem and I read that somewhere in card.io repo

**Comment by jagan27101986** [comment]
So has someone able to resolve that issue ?.

**Comment by ba0f3** [comment]
card-io/card.io-iOS-SDK#20

**Comment by afish8ben** [comment]
Still unable to scan an AmEx card within my application, the card in question has embossed silver letters and the correct digits appear on the camera, just no data gets passed back to my application.

Edit: Card itself only has 15 digits for the card number.  Does Card.IO not  look at the bin range during the scan process and do validation according to card type?
